src/main/java/entities/User.java

```diff
@@ -1,5 +1,66 @@
 package entities;
 
-abstract class User {
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+
+abstract class User extends Account{
+
+    private Profile profile;
+    private ArrayList<User> blocked = new ArrayList<>();
```

Comment:
`private List<User> blocked = new ArrayList<>();` Sorry, I forgot to change this to List<User>. Could you please modify it for me? Thx